HW10-hibernate/src/main/java/ru/otus/core/model/User.java

```diff
@@ -0,0 +1,98 @@
+package ru.otus.core.model;
+
+
+import javax.persistence.*;
+import java.util.List;
+
+@Entity
+@Table(name = "users")
+public class User {
```

Comment:
Или добавить `"DataSet"`

```suggestion
public class UserDataSet {
```

или у всех классов убрать `DataSet` -- `User`, `Address`, `Phone`